src/app.ts

```diff
@@ -19,24 +19,42 @@ import channels from './channels';
 import { HookContext as FeathersHookContext } from '@feathersjs/feathers';
 import authentication from './authentication';
 import mongoose from './mongoose';
+
 // Don't remove this comment. It's needed to format import lines nicely.
 
 const app: Application = express(feathers());
-export type HookContext<T = any> = { app: Application } & FeathersHookContext<T>;
+export type HookContext<T = any> = {
+  app: Application;
+} & FeathersHookContext<T>;
 
 // Load app configuration
 app.configure(configuration());
 // Enable security, CORS, compression, favicon and body parsing
-app.use(helmet({
-  contentSecurityPolicy: false
-}));
+app.use(
+  helmet({
+    contentSecurityPolicy: false
+  })
+);
 app.use(cors());
 app.use(compress());
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 app.use(favicon(path.join(app.get('public'), 'favicon.ico')));
-// Host the public folder
-app.use('/', express.static(app.get('public')));
+
+// Configure Swagger API
+app.configure(
+  swagger({
+    docsPath: '/swagger',
+    uiIndex: true,
+    specs: {
+      info: {
+        title: 'EddieCommunityApi',
+        description: "A Web API to access the community's Discord Server info.",
```

Comment:
we should have single quotes here too for consistency. By running our `lint` script this should be fixed 👍